### PR TITLE
add x button

### PR DESCRIPTION
**Pull request by lambtron**
image

click on it will remove the top bar. 

top bar will re appear on reloading the page.

### CODE REVIEW

File: components/HelloBar.tsx [hunk 1]
```diff
@@ -10,6 +10,7 @@ export function HelloBar(props: {
 }) {
   return (
     <div class={tw`text-center bg-black text-white`}>
+      <span class={tw`float-left cursor-pointer py-2 pl-4`} onclick="this.closest('div').remove();">✕</span>
```
[the remark this answers is not in the document]
Comment (lambtron) [reply components/HelloBar.tsx:13]: haha island is definitely the fresh way to do it... but i'm lazy and figured a one in-line html/js hack could work. if i have time i can turn it into the world's tiniest island.

[the remark this answers is not in the document]
Comment (crowlKats) [reply components/HelloBar.tsx:13]: if not, thats fine, we can leave it as-is. i think cases like these inline js ike that is perfect, and island might maybe be a bit overkill even

**Comment by kidonng** [comment]
> top bar will re appear on reloading the page.

That makes the button less useful IMO. Maybe something like this:

```js
const onClick = () => localStorage.setItem('hellobar', content)
const onLoad = () => { if (localStorage.getItem('hellobar') === content) return }
```

**Comment by lucacasonato** [comment]
> > top bar will re appear on reloading the page.
> 
> That makes the button less useful IMO. Maybe something like this:
> 
> ```js
> const onClick = () => localStorage.setItem('hellobar', content)
> const onLoad = () => { if (localStorage.getItem('hellobar') === content) return }
> ```

In that case it should be a cookie so it don't FOUC on SSR.

**Comment by crowlKats** [comment]
Closing as not applicable anymore

**Pull request closed without merging**